frontend/src/MuiForms/EntitySelectorBeta.tsx

```diff
@@ -27,11 +29,14 @@ export default function EntitySelectorBeta(props: EntitySelectorBetaProps) {
   const entities = useMemo(() => {
     if (!users) return []
 
-    return users.map((user) => user.id)
+    const userKind = users.find((userGroup) => userGroup.kind === 'user')
```

Comment:
Again not sure about `userKind` as a name. `userGroup` makes more sense, but then you'd need to rename the param inside the `find` to maybe something like `userGrp` or `usrGroup`?